### DISCUSSION TITLE
Pyvista element selection in user side within a web app

### DISCUSSION THREAD

**Discussion by ArioYD** [community/community#38229] [Other Feature Feedback, Questions, & Ideas]
I am developing a web app (engineering software) using python-Flask. User input and analysis outputs are visualized in the web app using pyvista.

The 3D visualization model is a pyvista file in html that has been generated in the server side and is shown  in an ifram html tag in user side.

Since the number of elements (lines, faces, and volumes) within the model is relatively high, i want to give the user the capability to select an element or a group of elements on his/her web browser (user side) and then delete them or assign a new property to them. 

How can we do that? 

Looking for your help on this matter.

Regards
Ario

**Comment by ossanna16** [comment]
Hi @ArioYD,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Flask. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
